lib/krane/deploy_task.rb

```diff
@@ -273,6 +275,7 @@ def validate_configuration(prune:)
 
       confirm_ejson_keys_not_prunable if prune
       @logger.info("Using resource selector #{@selector}") if @selector
+      @logger.info("Can deploy a subset of resources") if @selector && @selector_as_filter
```

Comment:
Perhaps something a bit more explicit:

```suggestion
      @logger.info("Only deploying resources that match filter: #{@selector_as_filter}") if @selector && @selector_as_filter
```